### DISCUSSION TITLE
OpenClash 全局模式无效

### DISCUSSION THREAD

**Discussion by palemoky** [vernesong/OpenClash#2025] [Q&A] [answered]
我发现 OpenClash 有多个地方能够设置代理模式，但在一处设置了代理模式后，其它地方的状态都不一致
1. 运行状态的代理模式
![运行状态的代理模式](https://user-images.githubusercontent.com/21208635/152805409-ed58782b-6c8f-493d-a2e3-0ee0fda7683a.png)

2. 全局设置中的代理模式
![全局设置中的代理模式](https://user-images.githubusercontent.com/21208635/152805534-c67239fa-1942-4ff2-adab-017cdbd7ff5b.png)

3. 控制面板中的代理模式
![控制面板中的代理模式](https://user-images.githubusercontent.com/21208635/152805659-55012a87-3266-41ef-837e-f6fee198046c.png)

我全部调整为全局代理模式后，在规则代理模式下能打开的 Google 在全局模式下反而打不开了，ip138 显示的 IP 也不是代理 IP。配置文件软路由和电脑都用的同一份，电脑的全局是没有问题的，ShadowSocksR Plus+ 的全局模式也是正常的，请问 OpenClash 是我哪里配置有问题吗？

**Comment by vernesong** [comment, marked as answer]
全局模式要在控制面板选节点

**Reply by palemoky** [reply]
我在控制面板已经选择了节点
![image](https://user-images.githubusercontent.com/21208635/153315372-4e35dd4c-ecb5-47c3-a414-19da3beffb62.png)

然后开启了临时全局模式开关
![image](https://user-images.githubusercontent.com/21208635/153315412-afc9fa5a-c110-4516-9871-8b4009942142.png)

在 IP138 测到的还是我的本机 IP，请问是我哪里设置有问题吗？

**Comment by vernesong** [comment]
日志？

**Reply by palemoky** [reply]
谢谢，我分析下日志看看

**Comment by yangcao111** [comment]
一样的问题等待解答

**Reply by palemoky** [reply]
你把日志开到 debug，然后看看是不是所有规则命中了代理模式，如果没有那可能你配置文件有问题，我最后发现是自己的配置问题。

**Comment by yangcao111** [comment]
全局模式默认好像是直连，要手动选节点
